File: easy/find_the_difference.py
from collections import Counter

def find_the_difference(s: str, t: str)-> str:
	counts = Counter(s)
	for char in t:
		counts[char] = counts.get(char, 0) - 1
		if counts.get(char, 0) < 0:
			return char
	return ""

File: easy/find_the_difference.go
package easy

import (
	core "github.com/wearypossum4770/dark-coding-challenges/dark_coding_challenges"
)

func FindTheDifference(s string, t string) byte {
	counts := core.CharacterFrequency(s)
	for _, char := range t {
		counts[char]--
		if counts[char] < 0 {
			return byte(char)
		}
	}
	return ' '
}
